src/content/en/updates/2019/03/kv-storage.md

```diff
@@ -144,12 +153,13 @@ If you're familiar with using native JavaScript modules in browsers, you
 probably know that (at least up until now) importing anything other than a URL
 will generate an error. And `std:kv-storage` is not a valid URL.
 
-So that raises the question: _do we have to wait until all browsers support
-built-in module before we can use it in our code?_
+So that raises the question: _do we have to wait until all browsers support a
+built-in module before we can use it in our code?_ Thankfully, the answer is no!
```

Comment:
So, that's already kinda true today for other features, with the help of polyfills.
But the great thing about built-ins is that they are:
a) polyfillable by definition
b) will only load the polyfill when needed (in browsers that support built-ins/import-maps), without requiring any complex server side shenanigans.

You may want to emphasize those points here.

Reply:
The difference here though is that in the example shown, built-in modules are imported using syntax that will break in most browsers. And normally there's no way to polyfill unsupported syntax, so I do think it's different in this case.

As for the other points, I do I cover them in the section that follows, but I'd be happy to expand on them in a separate PR if you feel it's necessary (I want to get these changes in soon if possible).